runners/maven-plugin/src/main/kotlin/DokkaMojo.kt

```diff
@@ -243,7 +243,7 @@ abstract class AbstractDokkaMojo : AbstractMojo() {
                 if (sourceSet.moduleDisplayName.isEmpty()) logger.warn("Not specified module name. It can result in unexpected behaviour while including documentation for module")
             },
             pluginsClasspath = getArtifactByAether("org.jetbrains.dokka", "dokka-base", dokkaVersion) +
-                    dokkaPlugins.map { getArtifactByAether(it.groupId, it.artifactId, it.version) }.flatten(),
+                    dokkaPlugins.map { getArtifactByAether(it.groupId, it.artifactId, it.version ?: dokkaVersion) }.flatten(),
```

Comment:
Do you think it makes sense for users that we automatically append **our** dokka version to artifacts? I can imagine that this leads to confusion and unwanted behaviour for users?

Reply:
It's not every artifact you can imagine. These are only dokka plugins. If you know certain version of plugin, you can specify it explicitly. However, if you don't know what plugin version should be linked, it will try to match plugin based on dokka-base version. It's useful for example for javadoc artifacts. I don't know what version user would like to use, I cannot pass it other way, but I can implicitly assume its version on dokka-base version